installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -1,28 +1,37 @@
-[id='proc-install-rhdh-ocp-operator_{context}']
-
+[id="proc-install-rhdh-ocp-operator_{context}"]
 = Deploying {product} on {ocp-short} using the Operator
 
-As a developer, you can install {product} on your OpenShift Container Platform using the {product} Operator.
+As a developer, you can deploy a {product} instance on {ocp-short} by using the *Developer* perspective in the web console.
 
 .Prerequisites
-* Your administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
-* You have created a project in {ocp-short} for your {product} instance. For more information about creating a project in OpenShift, see link:https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects[Red Hat OpenShift documentation].
+
+* A cluster administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
 
 .Procedure
-. From the *Developer* perspective on the {product-short} web console, click *+Add*.
+
+. Create a project in {ocp-short} for your {product} instance, or select an existing project.
++
+[TIP]
+====
+For more information about creating a project in {ocp-short}, see link:https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects[Creating a project by using the web console] in the {ocp-brand-name} documentation.
+====
+. From the *Developer* perspective on the {ocp-short} web console, click *+Add*.
```

Comment:
We might want to use consistent wording to refer here to the *+Add*  tab, and later to the *Topology* tab.

![03- in the project, in the +Add tab Screenshot_+Add · Red Hat OpenShift — Mozilla Firefox_20240419_103959](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/243761/0981286f-eb04-4556-ad8d-63e3798d395b)

Reply:
(Clarity nitpick, sorry)

Reply:
I think it might make more sense to try to make our docs more consistent with the terms used in the core OCP docs, where it's [generally "**+Add** page" AFAIK](https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#adding-to-a-project_projects). "Tab" means something specific in the console because there are pages that have actual tabs ([example](https://docs.openshift.com/container-platform/4.14/applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.html#odc-monitoring-your-project-metrics_monitoring-project-and-application-metrics-using-developer-perspective)).

I think I'll leave this as is for now and we can discuss it with the team and decide on some guidelines to update all the instances going forward?